components/ResourceCreation/TestResourceCreation.tsx

```diff
@@ -115,6 +140,14 @@ function TestResourceCreation() {
     return undefined;
   };
 
+  const getConfirmationModalText = (id: string | null) => {
```

Comment:
Nitpick but maybe we should change `id` to `resourceId` so that it's clear that the id refers to the resource and not the selected patient (and to be consistent with the `getConfirmationModalText` in `PatientCreation.tsx` that uses `patientId` as the input rather than `id`)